spec/rmagick/draw/interline_spacing_writer_spec.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.describe Magick::Draw, '#interline_spacing=' do
+  let(:draw) { described_class.new }
+
+  it 'assigns without raising an error' do
+    expect { draw.interline_spacing = 1 }.not_to raise_error
+  end
```

Comment:
Not sure if this is now the moment to do it but maby we could also add a test for an invalid parameter? And also in some other tests?

Reply:
@dlemstra I think I'd like to leave off adding new specs to the next phase of the process. That way we can make sure we first get through splitting them up in a reasonable amount of time. After this, the next step will be going through all of the specs with with a fine toothed comb, cleaning them up and adding new ones.